File: php/uri1012.php
<?php

$pi = 3.14159;
$valor = explode(" ", fgets(STDIN), 3);

$areaTriangulo = number_format($valor[0] * $valor[2] / 2, 3, '.', '');
$areaCirculo = number_format(pow($valor[2], 2) * $pi, 3, '.', '');
$areaTrapezio = number_format(($valor[0] + $valor[1]) * $valor[2] / 2, 3, '.', '');
$areaQuadrado = number_format(pow($valor[1], 2), 3, '.', '');
$areaRetangulo = number_format($valor[0] * $valor[1], 3, '.', '');

echo "TRIANGULO: {$areaTriangulo}\n";
echo "CIRCULO: {$areaCirculo}\n";
echo "TRAPEZIO: {$areaTrapezio}\n";
echo "QUADRADO: {$areaQuadrado}\n";
echo "RETANGULO: {$areaRetangulo}\n";

?>

File: java/uri1012.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Locale;

public class uri1012 {
 
    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);
        
        String linha = sc.nextLine();
        String[] elemento = linha.split(" ");

        double pi = 3.14159;
        double A = Double.parseDouble(elemento[0]);
        double B = Double.parseDouble(elemento[1]);
        double C = Double.parseDouble(elemento[2]);

        double areaTriangulo = A * C / 2;
        double areaCirculo = pi * Math.pow(C, 2.0);
        double areaTrapezio = (A + B) * C / 2;
        double areaQuadrado = Math.pow(B, 2.0);
        double areaRetangulo = A * B;

        System.out.printf("TRIANGULO: %.3f%n", areaTriangulo);
        System.out.printf("CIRCULO: %.3f%n", areaCirculo);
        System.out.printf("TRAPEZIO: %.3f%n", areaTrapezio);
        System.out.printf("QUADRADO: %.3f%n", areaQuadrado);
        System.out.printf("RETANGULO: %.3f%n", areaRetangulo);
    }
 
}